recipes/database.rb

```diff
@@ -0,0 +1,12 @@
+REPO = "https://raw.githubusercontent.com/spartansystems/spartan-composer-recipes/master/files/".freeze
+gem "pg"
```

Comment:
Is the sqlite gem already installed by another recipe? Would it hurt to have it in two recipes to make sure that each recipe that depends on it will get it even if the others get refactored?